src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -223,6 +223,28 @@ public Application.MemoryStatus MemoryStatus
         }
     }
 
+    /**
+      * @brief Event arguments that passed via NUIApplicationDeviceOrientationChanged signal
+      */
+    internal class NUIApplicationDeviceOrientationChangedEventArgs : EventArgs
+    {
+        private Application.DeviceOrientationStatus status;
+        /**
+          * @brief Application - is the application that is being affected when the device orientation is changed.
```

Comment:
same as above

Reply:
I see, I will update all of cases in this file.